src/lib/stores/filter_data.js

```diff
@@ -0,0 +1,3 @@
+import {writable} from 'svelte/store';
+
+export const filter_data = writable();
```

Comment:
I used a store because i wanted to avoid the more complex syntax of passing data to parent components. And i thought as people only visit one side where they filter stuff, using the same store would work. but now that i think of it, i dont know what happens when people have two tabs with filters open.